lib/sandbox/sandbox_unsupported.go

```diff
@@ -0,0 +1,50 @@
+// +build !linux
+
+package sandbox
+
+import (
+	"fmt"
+	"os"
+)
+
+func isNSorErr(nspath string) error {
+	return nil // TODO(vbatts) ... really not sure ...
+}
+
+func newNetNs() (*NetNs, error) {
+	return &NetNs{}, fmt.Errorf("netns is not implemented for this platform")
+}
+
+func getNetNs(path string) (*NetNs, error) {
+	return &NetNs{}, fmt.Errorf("netns is not implemented for this platform")
+}
+
+// NetNs handles data pertaining a network namespace
+// for non-linux this is a noop
+type NetNs struct {
+	symlink *os.File
```

Comment:
We can drop this private property from the no-op placeholder.

Reply:
I remember adding this, because of the fields and functions by `os.File` are required, but I'll look into this

Reply:
> I remember adding this, because of the fields and functions by `os.File` are required…

Ah, you're right.  For example, you look at it [here][1].  So I'm fine with this line as it stands.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1439/files#diff-a4c4c99001006b51c987c21d1d410cebR338